### PR TITLE
QUIZ 78: accordion highlighting bug

### PR DESCRIPTION
**Pull request by alexgirard**
Now selects the accordion no matter where you click on it

budfix2

### CODE REVIEW

File: src/@types/state.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export type SetState<T> = React.Dispatch<React.SetStateAction<T>>;
```
Comment (heathermusson) [inline src/@types/state.d.ts:1]: Why was this moved?

**Review by heathermusson — APPROVED** [review]

File: src/@types/state.d.ts (hunk 1 above)
Comment (alexgirard) [reply src/@types/state.d.ts:1]: ngl i think this is a rebasing problem (since I made this pr off of bens branch that wasnt merged in yet)
